File: src/classes/api/req_res/add-text-response.ts
// ˅
'use strict';

import { Text } from '@/classes/datas/text';
import { GkillAPIResponse } from '../gkill-api-response';

// ˄

export class AddTextResponse extends GkillAPIResponse {
    // ˅
    
    // ˄

    added_text: Text;

    constructor() {
        // ˅
        super()
        this.added_text = new Text()
        // ˄
    }

    // ˅
    
    // ˄
}

// ˅

// ˄

File: src/server/gkill/api/req_res/add_text_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type AddTextResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	AddedText *reps.Text `json:"added_text"`
}
